pkg/volume/gcepd/gce_pd.go

```diff
@@ -412,35 +410,31 @@ func (b *gcePersistentDiskMounter) SetUpAt(dir string, mounterArgs volume.Mounte
 	if err != nil {
 		notMnt, mntErr := b.mounter.IsLikelyNotMountPoint(dir)
 		if mntErr != nil {
-			klog.Errorf("IsLikelyNotMountPoint check failed: %v", mntErr)
-			return err
+			return fmt.Errorf("failed to mount: %v. Cleanup IsLikelyNotMountPoint check failed: %v", err, mntErr)
```

Comment:
not your change. but I am wondering whether it is necessary to check IsLikelyNotMountPoint multiple times. If mount failed, is it better we just try to unmount (maybe a few times) and then try to remove dir.

Reply:
This is outside of the scope of this PR and I won't change it. Feel free to create an issue :)

Reply:
I opened an issue here https://github.com/kubernetes/kubernetes/issues/83515